pkgs/applications/audio/spotify-player/default.nix

```diff
@@ -26,15 +29,25 @@ rustPlatform.buildRustPackage rec {
   nativeBuildInputs = [
     pkg-config
     cmake
+  ] ++ lib.optional stdenv.isDarwin [
+    rustPlatform.bindgenHook
```

Comment:
I think it is alright to include `bindgenHook` for non-darwin too - for future-proofing.

Reply:
Seems like it wasn't required for the Linux build, but will add it.